rax-swiper 启动时报错

```bash
./node_modules/_rax-swiper@0.2.0@rax-swiper/es/web/init-swiper.js Module not found: Error: Package path ./swiper-bundle.min.css is not exported from package /Users/kongshan/Projects/txd-app-mini-detial/node_modules/_rax-swiper@0.2.0@rax-swiper/node_modules/swiper (see exports field in /Users/kongshan/Projects/txd-app-mini-detial/node_modules/_rax-swiper@0.2.0@rax-swiper/node_modules/swiper/package.json) 
```

引用的 swiper.js 更新到了 8.3.1，在 pacakge.json 中的 exports 改了路径 目前是这样 "exports": { // ... "./bundle": "./swiper-bundle.esm.js", // ... }

--- a/packages/rax-swiper/CHANGELOG.md
+++ b/packages/rax-swiper/CHANGELOG.md
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.2.2-beta.0
+
+- [fix] import css of swiper not found.
+
 ## 0.2.0
 
 - [feat] update swiper version to 8.x

--- a/packages/rax-swiper/package.json
+++ b/packages/rax-swiper/package.json
@@ -1,7 +1,7 @@
 {
   "name": "rax-swiper",
   "author": "rax",
-  "version": "0.2.0",
+  "version": "0.2.2-beta.0",
   "description": "Swiper component for Rax.",
   "license": "BSD-3-Clause",
   "main": "lib/index.js",

--- a/packages/rax-swiper/src/web/init-swiper.js
+++ b/packages/rax-swiper/src/web/init-swiper.js
@@ -2,7 +2,7 @@
 import { Swiper, Autoplay, Pagination } from 'swiper';
 import { needsNavigation, needsPagination, needsScrollbar } from './utils';
 
-import 'swiper/swiper-bundle.min.css';
+import 'swiper/css/bundle';
 
 Swiper.use([Autoplay, Pagination]);
 function initSwiper(swiperParams) {

